fpga/src/main/scala/ld/LineDrawing.scala

```diff
@@ -99,6 +97,7 @@ class LineDrawing(
       state := init3
     }
     is(init3) {
+      io.writeEnable := true.B
```

Comment:
Hvorfor trengs denne writeEnable egentlig?

Reply:
Den sendes rett til framebufferet. Aka sier om vi skal skrive verdien som linedraweren har satt for x og y

Reply:
Ja, fårstår det, men hvorfor sendes den i Init? Burde ikke den bare være høy i draw?

Reply:
Hmm, tenkte det var fordi den måtte være høy en sykel før draw siden x og y initiellverdier settes i init, men ser nå at x og y er stable i to sykler (ref. waveformen det er bilde av) så kunne kanskje holdt med kun draw ja :thinking:. Det var vel en grunn til at jeg satte den der originalt (håper jeg) men ble litt usikker nå. Kan evt. teste i hw. 

Men i værste fall skriver vi bare samme verdien to ganger sånn som det er nå :man_shrugging:

Reply:
Det var i alle fall en grunn til at jeg la til `init3` (som i teorien ikke skal trengs) men kan være at den også er overføldig nå, om vi har fått ryddet opp i litt andre ting i denne modulen (som registrene med `updX` og `updY`). Så kan være vi kan fjerne den også.